conflux/src/endpoint/conflux.ts

```diff
@@ -0,0 +1,84 @@
+import { Requester, Validator } from '@chainlink/external-adapter'
+import { AdapterRequest, AdapterResponse } from '@chainlink/types'
+import { Conflux } from 'js-conflux-sdk'
+import { ethers } from 'ethers'
+import { Config } from '../config'
+
+const sendFulfillment = async (
+  provider: any,
+  from: any,
+  to: string,
+  dataPrefix: string,
+  functionSelector: string,
+  value: number,
+) => {
+  const dataPrefixBz = ethers.utils.arrayify(dataPrefix)
+  const functionSelectorBz = ethers.utils.arrayify(functionSelector)
+  const valueBz = ethers.utils.zeroPad(ethers.utils.arrayify(Number(value)), 32)
+  const data = ethers.utils.concat([functionSelectorBz, dataPrefixBz, valueBz])
+
+  const tx = {
+    from: from,
+    to: to,
+    data: ethers.utils.hexlify(data),
+    gas: 500000,
```

Comment:
Not very fan of setting the gas limit manually. It seems you could use:
```js
const estimate = await provider.estimateGasAndCollateral({ to, value, data });
const tx = {
    // ...
    gas: estimate.gasUsed,
    // ...
}
````

Reply:
It's seems chainlink's contract require the gas bigger than 400000, so we set the gas manually here